feat(rust): environment command & moved text

Neruelin opened the pull request:
## Current behavior

Information about Environment Variables is displayed in the footers of other commands taking up a large space.

## Proposed changes

Created a new 'environment' CLI command to display the Environment Variables information separately, and removed the moved text from the other CLI footer.

Closes  issue #4928

## Checks

- [✅] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [✅] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [✅] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [✅] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

mrinalwadhwa commented:
@Neruelin Thank you for spending time on this, while we review please

- [ ] Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will make the commit as verified.

Neruelin commented:
> @Neruelin Thank you for spending time on this, while we review please
> 
> * [x]  Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.
> 
> The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will make the commit as verified.

Created the contributors PR
https://github.com/build-trust/ockam-contributors/pull/216

Neruelin committed:
feat(rust): environment command & moved text

Adding environment CLI command to display environment variable info separately rather than
included in other command output.
Amended with envtext adding final newline

adrianbenavides commented on implementations/rust/ockam/ockam_command/src/environment/mod.rs:
```diff
@@ -0,0 +1,13 @@
+use clap::Args;
+
+const ENVTEXT: &str = include_str!("./static/envtext.txt");
+
```
You could add a short documentation line for the command like: `Show the Ockam environment variables`

adrianbenavides commented on implementations/rust/ockam/ockam_command/src/environment/static/envtext.txt:
```diff
@@ -0,0 +1,24 @@
+Environment Variables:
+
```
These two lines are not needed anymore.

adrianbenavides reviewed (dismissed):
Thanks for submitting another PR @Neruelin, just two small fixes before we can merge it 👍

Neruelin committed:
feat(rust): doc comment + trim static text

Added a comment explaining the purpose of the environment command.
Trimmed unnecessary lines from environment static text.

etorreborre committed:
style(rust): fix the formatting

adrianbenavides reviewed (approved):
LGTM! Congrats @Neruelin for landing your first PR at ockam 🥳
